Backend/Rooms/Repository/FileRepository/EquipmentTypeRepository.cs

```diff
@@ -45,5 +45,25 @@ public bool ExistsInSystem(string name)
             var allEquipmentTypes = stream.GetAll().ToList();
             return allEquipmentTypes.Any(item => item.Name== name);
         }
+
+        public bool ExistsInSystem(int id)
+        {
+            throw new NotImplementedException();
```

Comment:
implement all not implemented method

Reply:
even better delete all file repositories that are causing problems and resolve conflicts